testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -75,4 +75,13 @@ export default class InstitutionHelperActions {
   getUniqueId() {
     return v4();
   }
+
+  getRandomInstitutionCode() {
```

Comment:
Location Institution codes must be unique for a specific institution otherwise the insert on DB will fail. Even with a small possibility, the test can ramdomically fail.

Reply:
Updated it to check the existing institution codes, until we find out the way to control the data